src/components/navbar/navLinkSection/NavLinkSection.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classes from './NavLinkSection.module.scss';
+import PropTypes from 'prop-types';
+
+import { openNewsletterSignupModal } from '../.././../utils/functions';
+
+const NavLinkSection = ({ navLinks, toggleDisplayNav }) => {
+    const handleNewsletterClick = () => {
+        if (toggleDisplayNav) {
+            toggleDisplayNav(false);
+        }
+        openNewsletterSignupModal(true);
+    };
+
+    const findNavOutput = ({ display, path, external, modal }) => {
+        switch (true) {
+            case path !== undefined:
+                return (
+                    <Link
+                        key={display}
+                        className={classes.link}
+                        to={path}
+                        onClick={() => toggleDisplayNav()}
```

Comment:
This strangely is throwing a no function error... I am unsure if it is just me?
![Screenshot 2020-09-07 at 10 41 25](https://user-images.githubusercontent.com/53180713/92373737-b0f9bf80-f0f6-11ea-8b46-c48bb32a19ca.png)

I do have a workaround which solved it on my repo for all links, don't call the function as it doesn't take a parameter (or should it?)

```suggestion
                        onClick={toggleDisplayNav}
```

Reply:
doing it this way works for me in max width, but breaks on 800 width, I will have to work on it sometime wednesday